apps/dg/dg-docassembly/dg-docassembly.yaml

```diff
@@ -8,7 +8,9 @@ spec:
   values:
     java:
       useInterpodAntiAffinity: true
-      replicas: 2
+      replicas: 2  # minReplicas will default to this value
+      autoscaling:
```

Comment:
Why not autoscale by default?

Reply:
We don't need autoscaling on non-prod environments?